sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log azure-storage-blob
 
+## Version XX.X.X (XXXX-XX-XX)
```

Comment:
Should this be updated?

Reply:
This won't get merged into master, so I'm not sure it matters. If it does, I'll do a separate PR on top of this so I don't have to restart CI for a readme change.

Reply:
Right now this doesn't matter but we are working on better improvements to pull the changelog information into the release process as well as validating it so it will matter more in the future.